app/themes/dashboard/ui/styles/theme.less

```diff
@@ -43,3 +43,11 @@ z-index: 10;
 .pre-xml {
   background-color: #ecf0f5;
 }
+
+.nav-pills>li>a {
+  border-radius: 4px !important;
+}
+
+.nav-pills>li.active>a {
+  background-color: #384246 !important;
```

Comment:
I had to use !important to override a theme (adminLTE.less) that was imported inside theme.less. Based on another comment, I'll remove the rounding for the buttons.